Detections/3pNetworkDetetions/PhishiglinkExecutionObserved.yaml

```diff
@@ -0,0 +1,83 @@
+id: 2fed0668-6d43-4c78-87e6-510f96f12145
+name: Phishing link click observed in 3P Network device
+description: |
+  'sucessfull Phishing link accessed via "Palo Alto Networks","Fortinet","Check Point","Zscaler" devices.'
```

Comment:
Let's make this more detailed to really describe what we are doing here, and why customers might want to deploy this content.

Reply:
Changed the description.